tools/esp-api/espapi.cpp

```diff
@@ -264,22 +261,28 @@ void EspDef::describeAllServices(ostream &out)
         const char* serviceName = currentService.queryName();
         out << serviceName << " ";
         Owned<IPropertyIterator> propIter = currentService.getProps();
-        out << "[";
         bool first = true;
         ForEach(*propIter)
         {
             if(streq(propIter->getPropKey(), "name"))
             {
                 continue;
             }
-            if(!first)
+            if(first)
+            {
+                out << "[";
```

Comment:
The leading space, currently added unconditionally on line 262, should be included in this string. It should only be added if meta data is going to be output.